README.md

```diff
@@ -59,10 +60,33 @@ The Studio connects to Sanity Content Lake, which gives you hosted content APIs
 Use the Deploy Button below. It will let you deploy the starter using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-sanity-example) as well as connect it to your Sanity Content Lake using [the Sanity Vercel Integration][integration].
 
 [![Deploy with Vercel](https://vercel.com/button)][vercel-deploy]
+[![Deploy with Netlify](https://www.netlify.com/img/deploy/button.svg)][netlify-deploy]
+
+If you are using **Netlify**, then you will need to:
+
+- Create a new [Sanity project][sanity-create] and at the end note the values for `--project` and `--dataset`:
```

Comment:
I've based myself for this on a detail that we had a while back and was removed because [Mikolaj mentioned that we probably want to remove it altogether.](https://sanity-io.slack.com/archives/C045U0L0D43/p1666620842515179)What would be the best instruction to get people to create the project? 

Should it be from here: https://www.sanity.io/docs/getting-started-with-sanity-cli and then explain how to get the dataset and project id from manage? @kmelve @evenwestvang

Reply:
Alternatively we could have another `.md` file that we link to?

Reply:
I'm not sure about adding a new .md 🤔 but I've updated the text again to create a project with CLI